common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -475,6 +475,8 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     REPL_DUMPDIR_TTL("hive.repl.dumpdir.ttl", "7d",
         new TimeValidator(TimeUnit.DAYS),
         "TTL of dump dirs before cleanup."),
+    REPL_DUMP_COPY_DATA("hive.repl.dump.copydata", true,
```

Comment:
what happens when both REPL_DUMP_COPY_DATA and REPL_DUMP_METADATA_ONLY is set to true? which one takes precedence?